ssrc_config/config_hitl_eth_gzsim_fw.txt

```diff
@@ -0,0 +1,118 @@
+# Default parameter set for HITL with ethernet Gazebo Sim connection on Skywalker X8 FW
+# [ type: hitl_eth_gzsim ]
+
+param set SYS_AUTOSTART 4421
+param set SYS_AUTOCONFIG 0
+
+####################################
+# general HITL configuration
+####################################
+
+# Set HITL related flag
+param set SYS_HITL 1
+param set MAV_HITL_SHOME 1
+param set SENS_EN_GPSSIM 1
+param set SENS_EN_BAROSIM 1
+param set SENS_EN_MAGSIM 1
+param set SENS_EN_ARSPDSIM 1
+
+# Disable safety switch
+param set COM_PREARM_MODE 0
+param set CBRK_IO_SAFETY 22027
+
+# Disable RC loss failsafe check
+param set NAV_RCL_ACT 0
+
+# Disable RC input requirement
+param set COM_RC_IN_MODE 1
+
+# EKF2
+param set EKF2_MULTI_IMU 3
+
+# Sensor
+param set CAL_ACC0_ID  1310988
+param set CAL_GYRO0_ID 1310988
+param set CAL_ACC1_ID  1310996
+param set CAL_GYRO1_ID 1310996
+param set CAL_ACC2_ID  1311004
+param set CAL_GYRO2_ID 1311004
+param set CAL_MAG1_ID 197388
+param set CAL_MAG1_PRIO 50
```

Comment:
What is the reason to set these calibration values? If physical sensors are not in use in HITL mode, then does these have any effect?

Reply:
these were the residual from the `config_hitl_eth_gzsim.txt` . I remove them now